llm_demo/orchestrator/langchain_tools/langchain_tools_orchestrator.py

```diff
@@ -70,6 +73,12 @@ def initialize_agent(
         model: str,
     ) -> "UserAgent":
         llm = VertexAI(max_output_tokens=512, model_name=model, temperature=0.0)
+        # Create an llm that has slightly higher temp and higher max_output_token for retries
+        less_predictable_llm = VertexAI(
+            max_output_tokens=1024, model_name=model, temperature=0.2
```

Comment:
Shouldn't we just increase the max_output_tokens on the original?

Reply:
We totally can, I just didn't want to change anything on the happy path before NEXT.